test/required/requiredexample.proto

```diff
@@ -0,0 +1,14 @@
+package required;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+message RequiredExample {
+    required string theRequiredString = 1;
+    optional string theOptionalString = 2;
+    repeated string theRepeatedStrings = 3;
+}
```

Comment:
I just tried that.
This seems to only be an error on this branch.
So I suspect it has to do with the newly generated code.
I suspect we have found a bug :)